exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,25 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function shout(hello) {
+  return hello;
+}
```

Comment:
So the exercise asks for one function that converts the mentor name to upper case and the other one that creates a shouty greeting. So if I gave you a function:

```javascript
// Converts name to upper case
function shout(name) {
    return name.toUpperCase();
}
```
And asked you to create a shoutyGreeting such that I could call it like this:
```node
>>> shoutyGreeting("Daniel")
>>> "HELLO DANIEL"
```

How would you write it?

Reply:
Tried to figure this out but couldn't quite get it.

Reply:
I did by looking at an example online but still don't fully understand how it works.